processor/src/tests/mod.rs

```diff
@@ -96,13 +102,20 @@ async fn test_send<C: Coin + Clone>(coin: C, fee: C::Fee) {
     let signable = wallet
       .prepare_sends(1, vec![(wallet.address(), 10000000000)], fee)
       .await
-      .unwrap()
-      .1
-      .swap_remove(0);
+      .unwrap().1.swap_remove(0);
     futures.push(wallet.attempt_send(network, signable));
   }
+  
+  if futures.len() > 0 {
```

Comment:
This if shouldn't exist. There should always be a future.

It also somewhat-voids the test.